tests/test_delete_files_in_range.rs

```diff
@@ -167,3 +167,39 @@ fn test_delete_files_in_range_with_delete_range() {
     assert_eq!(it.key(), b"5");
     assert!(!it.next());
 }
+
+#[test]
+fn test_delete_files_in_ranges() {
+    let path = TempDir::new("_rust_rocksdb_test_delete_files_in_multi_ranges").expect("");
+    let path_str = path.path().to_str().unwrap();
+    let db = initial_data(path_str);
+
+    // Delete files in multiple overlapped ranges.
+    // File ["key0", "key2"], ["key3", "key5"] should have been deleted,
+    // but file ["key6", "key8"] should not be deleted because "key8" is exclusive.
+    let mut ranges = Vec::new();
+    ranges.push(Range::new(b"key0", b"key4"));
+    ranges.push(Range::new(b"key2", b"key6"));
+    ranges.push(Range::new(b"key4", b"key8"));
```

Comment:
why `key6` and `key7` still exist since they are in range `[key4, key8)`

Reply:
Let's see, we have three files: ["key0", "key2"], ["key3", "key5"], ["key6", "key8"].
The range ["key0", "key4") covers the first file, so the first file will be deleted.
The range ["key2", "key6") covers the second file, so the second file will be deleted.
However, the range ["key4", "key8") doesn't cover the third file because "key8" is exclusive.
So, the third file ["key6", "key8"] will not be deleted, "key6", "key7", and "key8" still exist.